README.md

```diff
@@ -1,13 +1,88 @@
+- Users can complete an item by selecting the radio button next to the item. Once selected a green checkmark appears and the item is crossed off the list. The number of items left updates to show how many are uncompleted
+- Users can complete all items by selecting the down arrow above the radio buttons when there are uncompleted todos
+- Once a task is completed it can be viewed in the All or Completed views
+- Users can uncheck all todos by selecting the down arrow above radio buttons when all radios are selected
+- Users can edit a todo item by double clicking the item, then selecting Enter on the keyboard to submit the change
+- Users can delete an item in the todo list by selecting the "X" next to the todo item when they hover over the todo item. Once deleted the count of items left is updated to show uncompleted tasks. The item is removed from all lists. 
+- User can select All. This will show all incompleted and completed todos
+- User can select Active. This will only show all uncompleted todos
+- User can select Completed. This will only show all completed todos
+- User can select Clear completed to delete all Completed tasks. All completed tasks will be immediately be removed from view
+- Once a todo is added, the user will always see the bottom banner even if there is no todo in the view (All, Active, Completed)
+- Refreshing the page starts a new state and the users can start again from scratch
+
+### Test plan
+#### Outline a complete testing strategy that covers unit tests, integration tests, and E2E tests.
+
+#### Describe how you would implement security testing within the testing lifecycle.
+
+#### Propose how automated testing can be integrated into existing CI/CD pipelines.
+
+#### Detail any tools and frameworks you would recommend for the testing process, including any specific for performance and load testing.
+
+#### Discuss how you would ensure the testing strategy aligns with agile development practices.
+
+### 2. Update/Create Playwright tests
+We've created two example tests in Playwright, but they could use your help. Can you help improve these existing tests, and write new ones from your test plan? You can find these existing tests in the tests folder of this repo. 
+
+Don't feel like you need to make every single test in your test plan, but we do want to see a good example of your coding. Start by making 6 tests, and then if you are out of time you can make comments of other tests you would have made. 
+
+We also want to see an example of your API testing so we added 3 tests in there using the SWAPI API. More info is in the comments of the todo-CRUD.spec.ts folder of the SWAPI API and what tests we want to see.
+
+To run Playwright you might need to install Playwright
+While in your project directory navigate to the tests folder and run `npx playwright install`
+To run the Playright UI run `npx playwright test --ui`
```

Comment:
This should be added above to the initial installation steps in my opinion on like line 34